jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -38,12 +39,19 @@
 
   @Nullable
   static War getWarTask(Project project) {
-    WarPluginConvention warPluginConvention =
-        project.getConvention().findPlugin(WarPluginConvention.class);
-    if (warPluginConvention == null) {
+
+    if (!project.getPlugins().hasPlugin(WarPlugin.class)) {
       return null;
     }
-    return (War) warPluginConvention.getProject().getTasks().findByName("war");
+
+    if (project.getPlugins().hasPlugin("org.springframework.boot")) {
```

Comment:
Can you add a test to verify this behavior too?

Reply:
Sure

Reply:
Done